packages/de1/src/converters/shotDescriptionHeader.ts

```diff
@@ -20,19 +20,30 @@ function decode(data: Buffer): ShotDescriptionHeader {
     .vars();
 }
 
-function encode(shotSettings: ShotDescriptionHeader): Buffer {
+function encode(header: ShotDescriptionHeader): Buffer {
   return new Serializer()
-    .char(shotSettings.headerV)
-    .char(shotSettings.numberOfFrames)
-    .char(shotSettings.numberOfPreinfuseFrames)
-    .char(shotSettings.minimumPressure * 16)
-    .char(shotSettings.maximumFlow * 16).buffer;
+    .char(header.headerV)
+    .char(header.numberOfFrames)
+    .char(header.numberOfPreinfuseFrames)
+    .char(header.minimumPressure * 16)
+    .char(header.maximumFlow * 16).buffer;
 }
 
-const converter: Converter<ShotDescriptionHeader> = {
+function validate(header: Partial<ShotDescriptionHeader>): boolean {
+  return Serializer.validate(header, {
+    headerV: Number(),
```

Comment:
Why `Number()` and not just `’number’`, matching `typeof` results?

Reply:
to be honest, that's probably just bad code